playbooks/roles/nginx/templates/nginx.conf

```diff
@@ -0,0 +1,74 @@
+user www-data;
+worker_processes 4;
```

Comment:
we might be able to leverage something like what's done in the current vhost files for Apache? 

https://github.com/dolph/keystone-deploy/blob/master/playbooks/roles/keystone/templates/apache/keystone.vhost#L1-L4